packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -78,6 +86,14 @@ export const searchChanged: Action<{ search: string }> = (
   state.dashboard.filters.search = search;
 };
 
+export const viewModeChanged: Action<{ mode: 'grid' | 'list' }> = (
```

Comment:
We don't use an object anymore when there's only 1 key

```suggestion
export const viewModeChanged: Action<'grid' | 'list'> = (
```

Reply:
This was never a standard, I perfer to send in keys, its easier to maintain in the future

Reply:
@christianalfoni & @CompuIves have commented this in one of my refactor PRs (can't find it back tho) that we should migrate to no object over time when there's only 1 key.